sql/updates/latest-dev.sql

```diff
@@ -1,3 +1,5 @@
 CREATE FUNCTION _timescaledb_functions.compressed_data_info(_timescaledb_internal.compressed_data)
-    RETURNS TABLE (algorithm name, has_nulls bool)
-    AS 'SELECT NULL,FALSE' LANGUAGE SQL STRICT IMMUTABLE SET search_path = pg_catalog, pg_temp;
+    RETURNS TABLE (algorithm cstring, has_nulls bool)
+    AS $$ SELECT ''::cstring,FALSE $$
+    LANGUAGE SQL STRICT IMMUTABLE
+    SET search_path = pg_catalog, pg_temp;
```

Comment:
Why do we have the search_path only in the dummy definition?

Reply:
Just following existing patterns.